catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/event/retrievestatus/DownloadsStatusEventPublisher.java

```diff
@@ -128,13 +128,19 @@ public void postRetrievalStatus(final ResourceResponse resourceResponse,
 
         // Add user information to the request properties.
         org.apache.shiro.subject.Subject shiroSubject = null;
+
         try {
             shiroSubject = SecurityUtils.getSubject();
         } catch (Exception e) {
```

Comment:
Is catching just a plain `Exception` bad practice since it may swallow errors we want exposed so they may be handled properly? If so, do we want to fix it on your ticket?